app/controllers/users_controller.rb

```diff
@@ -5,5 +5,6 @@ def index
 
   def show
     @user = User.find(params[:id])
+    @recent_posts = @user.recent_posts
   end
 end
```

Comment:
Here I initialize @recent_posts variable to store the return array from the recent_posts method.